### PR TITLE
Update spring cloud deployment and app table output to align with Azure portal

### PR DESCRIPTION
**Pull request by yuwzho**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Summary

Update table output for command, to align with portal display.
`az spring-cloud app list`
`az spring-cloud app show`
`az spring-cloud app deployment list`
`az spring-cloud app deployment show`

The new output should be look like 
image
and 
image

While the portal looks like
image

### General Guidelines

- [X] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [X] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

> N/A

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`. 

> Should be shipped with other feature together, no need to update version this time.

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S169

File: src/spring-cloud/azext_spring_cloud/_transformers.py [hunk 1]
```diff
@@ -31,17 +31,20 @@ def transform_app_table_output(result):
...
-                [x for x in instances if x['discoveryStatus'] == 'UP'])
-            item['Discovery Status'] = "UP( {} ), DOWN( {} )".format(
-                up_number, len(instances) - up_number)
+                [x for x in instances if x['discoveryStatus'].upper() == 'UP' or x['discoveryStatus'].upper() == 'OUT_OF_SERVICE'])
+            running_number = len(
+                [x for x in instances if x['status'].upper() == "RUNNING"])
+            item['Provisioning Status'] = item['properties']['activeDeployment']['properties']['provisioningState']
+            item['CPU'] = item['properties']['activeDeployment']['properties']['deploymentSettings']['cpu']
+            item['Memory'] = item['properties']['activeDeployment']['properties']['deploymentSettings']['memoryInGb']
+            item['Running Instance'] = "{}/{}".format(running_number, instance_count) if isStarted else "Stopped"
```
Comment (yungezz) [inline src/spring-cloud/azext_spring_cloud/_transformers.py:46]: it might cause problem if user replying on output json schema

Comment (yuwzho) [reply src/spring-cloud/azext_spring_cloud/_transformers.py:46]: There is no swagger change happened in backend side. My understanding is the json output should be align with the swagger model, and this file only take effects on table output. Thus if a user rely on the json output, should not be affected. Is it right?

**Comment by MyronFanQiu** [comment]
@yuwzho CI has been fixed. 😄

**Pull request merged**